canonical-web-and-design/vanilla-framework
# New vanilla website

## Done

- Remove old "Design" section from docs
- Extract Accessibility docs into their own file
- Rename Tab buttons to segmented control
- Add tab menu on the docs pages for accessibility and design guidelines.
- Docs tab menu settings are set on `component_tabs.yaml` file.

## QA

- Code has already been reviewed for each PR separately - this PR is the final QA
- Go to /docs check the tab menus for each pattern and layout

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

![image](https://user-images.githubusercontent.com/6387619/164236502-bfcb2105-36d3-48ea-87f5-b34e69c148f6.png)
